src/sentry/integrations/github/payload.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+
+import hashlib
+import hmac
+import logging
+import six
+
+from django.http import HttpResponse
+from django.utils.crypto import constant_time_compare
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from simplejson import JSONDecodeError
+from sentry import options
+
+from sentry.utils import json
+from sentry.api.base import Endpoint
+
+logger = logging.getLogger('sentry.webhooks')
```

Comment:
i would maybe name this something specific to github or at least integrations

Reply:
Yeah, take a look at slacks https://github.com/getsentry/sentry/blob/63c50ec68ed4fb2314e20323198dd384487feba7/src/sentry/integrations/slack/utils.py#L18